dojo/fixtures/benchmark_requirement.json

```diff
@@ -7492,7 +7492,7 @@
     "pk": 418,
     "fields": {
       "category": 27,
-      "objective_number": "1.6.1",
+      "objective_number": "1.6.2",
```

Comment:
This file is the wrong file, I guess you were looking for Chart.yaml instead to bump the chart number.
It seems the linting didn't catch it, so I'll have to look again. However, you'd want to bump the Chart version to 1.6.3.

Reply:
fixed